mino/minogrpc/certs/disk_test.go

```diff
@@ -0,0 +1,197 @@
+package certs
+
+import (
+	"crypto/tls"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/dela/core/store/kv"
+	"go.dedis.ch/dela/internal/testing/fake"
+	"go.dedis.ch/dela/mino"
+)
+
+func TestDiskStore_Store(t *testing.T) {
+	db, clean := makeDb(t)
+	defer clean()
+
+	cert := fake.MakeCertificate(t, 0)
+
+	store := NewDiskStore(db, fake.AddressFactory{})
+
+	err := store.Store(fake.NewAddress(0), cert)
+	require.NoError(t, err)
+
+	err = store.Store(fake.NewAddress(12), cert)
+	require.NoError(t, err)
+
+	cert, _ = store.InMemoryStore.Load(fake.NewAddress(0))
+	require.NotNil(t, cert)
+	cert, _ = store.InMemoryStore.Load(fake.NewAddress(12))
+	require.NotNil(t, cert)
+
+	other := NewDiskStore(db, fake.AddressFactory{})
+
+	cert, err = other.Load(fake.NewAddress(0))
+	require.NoError(t, err)
+	require.NotNil(t, cert)
+	cert, err = other.Load(fake.NewAddress(12))
+	require.NoError(t, err)
+	require.NotNil(t, cert)
```

Comment:
This test does not verify (if I understand correctly) that we get the "right" certificate.
I would create two certs and verify that the store doesn't (for some reason) mix up certificates 0 and 12.
Basically, ignoring the "address" argument would still make this part of the test pass, as I understand it.